src/guide/reviewing-and-sponsoring-a-change.md

```diff
@@ -0,0 +1,73 @@
+
+As a Sponsor, Contributors will look up to you for guidance to get their contributions into the project - your actions will determine whether Contributors will feel welcome and want to engage further with the project beyond their initial attempt, or not. Let's not lose enthusiastic, engaged and technically competent Contributors due to a lack of communication. If there is a request in your area of expertise and you can't address it, at least acknowledge receipt of the request and provide an estimate for when you'll be able to give it your attention. A frank explanation of your time constraints or commitments will be appreciated and respected.
+
+There are many different reasons to sponsor a change and depending on the situation the exact steps involved may differ significantly. An experienced developer hired into a larger team of OpenJDK developers is likely to get much of the required information by osmosis and will likely need less help from a sponsor than the enthusiast sitting at home with no prior experience of large scale software development. This text aims to highlight the steps involved in sponsoring a change but can't cover all possible scenarios. As always, common sense takes precedence where the assumptions made here doesn't apply to your use case.
+
+## Reviewing a change on GitHub
+
+Code reviews in OpenJDK happens on [GitHub](https://github.com/openjdk). This guide will hint at the most common steps to perform a review but it's not a complete documentation of all features. For official documentation see the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+
+When you arrive at a pull request there are a few different sections that you can browse through, Conversation, Commits, Checks, and Files changed. We will look at the first and the last of these.
+
+### Conversation
+
+The Conversation section is where all discussion around the PR happens. Note that this section is intended for technical discussion around the actual implementation of a change. This is not the place to discuss the appropriateness of the change or have overarching design discussions, that should happen on the appropriate [mailing lists].
+
+You can either reply to existing comments by using the text field beneath each comment, or add a new comment by scrolling to the bottom of the page and use the larger text field where it says "Leave a comment".
+
+Reviews take time for everyone involved. Your opinions are much appreciated as long as they are on-topic and of a constructive nature. A comment of the form "I don't like this" won't help anyone to improve the change. Instead express what it is that you don't like and give exmaples of how to do it differently. Always be polite and make sure you read through all previous comments before adding your own, the answer you are looking for may already have been given.
```

Comment:
Typo: exmaples

I would add a note to check for hidden comments and resolved issues, as they may contain information relevant to your own comment